recipes/open-webui/meta.yaml

```diff
@@ -0,0 +1,126 @@
+{% set name = "open-webui" %}
+{% set version = "0.5.7" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/open-webui-{{ version }}.tar.gz
+  sha256: 2b20a8533e4cd4e6cc8394c2928fa7c0206ee656cd1eeec87f36bddcc05c4466
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }},<3.13
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }},<3.13
+    - fastapi ==0.111.0
+    - uvicorn ==0.30.6
```

Comment:
FWIW, such strict versions regiment AFAIK is not very common (if not non-recommended) in conda and other distros. Typically it is mimimal and possibly maximal (if API breakage expected) being specified